index.ts

```diff
@@ -37,6 +37,7 @@ export function indent(element: HTMLTextAreaElement): void {
 	} else {
 		insert(element, '\t');
 	}
+	event.stopImmediatePropagation()
```

Comment:
This is the wrong place, all tests are failing. Did you test it on Refined GitHub?

Reply:
I did, and everything is working correctly, let try something else :)